builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -509,9 +514,31 @@ func (b *backend) pathGetRawIssuer(ctx context.Context, req *logical.Request, da
 		return nil, err
 	}
 
-	certificate := []byte(issuer.Certificate)
-
 	var contentType string
+	var certificate []byte
+
+	// tmp
+	responseHeaders := map[string][]string{}
+	if hasHeader(headerIfModifiedSince, req) {
+		before, err := sc.isIfModifiedSinceBeforeIssuerLastModified, ref, responseHeaders)
+		if err != nil {
+			return nil, err // errorResponse, nil or nil, err?
+		}
+		if before {
+			return &logical.Response{
```

Comment:
Wondering if we want to refactor to make a "not modified since" generic response minter? Just a thought :-)